example/lib/sample_linear_page.dart

```diff
@@ -92,14 +90,14 @@ class _SampleLinearPageState extends State<SampleLinearPage> {
                   child: LinearPercentIndicator(
                     width: 140.0,
                     fillColor: Colors.green,
-                    lineHeight: 14.0,
-                    percent: 0.5,
+                    lineHeight: 40.0,
```

Comment:
instead of modifying the current samples, can you add a new sample? (new linearpercentindicator child).